src/codegen/metadata/attributes.py

```diff
@@ -0,0 +1,130 @@
+attributes = {
+    'Channel': {
+         6145: {
+            'access': 'read-write',
+            'name': 'AI_ATTEN',
+            'resettable': True,
+            'type': 'float64',
+            'ctypes_data_type': 'ctypes.c_double',
+            'python_data_type': 'float',
+            'description': 'Specifies the amount of attenuation to use.',
+            'has_explicit_read_buffer_size': False,
+            'bitfield_enum':'N/A',
+            'is_list':False,
+            'calling_convention':'StdCall',
+            'c_function_name':'AIAtten',
+            'handle_parameters':
+            {
+                'task':{
+                    "accessor": "self._handle", 
+                    "ctypes_data_type": "lib_importer.task_handle", 
+                    "cvi_name": "taskHandle"
+                },
+                'channel':{
+                    "accessor": "self._name", 
+                    "ctypes_data_type": "ctypes.c_char_p", 
+                    "cvi_name": "channel"
+                }
+            },
+            'is_object': False,
+            'python_class_name':'AI_Channel'
```

Comment:
How much of this metadata will no longer be used if we switch to the generic attribute functions as proposed in https://github.com/ni/nidaqmx-python/pull/210 ?

Reply:
I see the `has_explicit_read_buffer_size` property may get obsolete once we do the implementation in #210, but other than that most of the other properties would still be needed for the property documentation and return type modifications after we make the c method call.

Reply:
The generic attribute functions take an explicit read buffer size when getting a string or array.

Example:
``` 
DAQmxErrChk (DAQmxGetChanAttribute(taskHandle,chan,DAQmx_PhysicalChanName,chan,256));
```